src/video.ts

```diff
@@ -22,6 +24,7 @@ const parseVideo = (element: BlockElement, atoms?: Atoms): Option<Video> => {
 	}
 
 	const { posterUrl, duration, assets, activeVersion } = atom.data.media;
+	const title = atom.data.media.title
```

Comment:
Can we just destructure title here from `atom.data.media` as we've done with the others on the line above?

Reply:
👍

Reply:
I thought we tried that 🤔

Reply:
For some reason it works now